modules/http/http-worker.c

```diff
@@ -160,7 +160,7 @@ _setup_static_options_in_curl(HTTPDestinationWorker *self)
     {
       curl_easy_setopt(self->curl, CURLOPT_FOLLOWLOCATION, 1);
       curl_easy_setopt(self->curl, CURLOPT_POSTREDIR, CURL_REDIR_POST_ALL);
-      curl_easy_setopt(self->curl, CURLOPT_REDIR_PROTOCOLS, CURLPROTO_HTTP | CURLPROTO_HTTPS);
+      curl_easy_setopt(self->curl, CURLOPT_REDIR_PROTOCOLS_STR, CURLPROTO_HTTP | CURLPROTO_HTTPS);
```

Comment:
Shouldn't this be `curl_easy_setopt(self->curl, CURLOPT_REDIR_PROTOCOLS_STR, "http,https");`?

Reply:
right. fixed.